lib/db/queries/getTestRuns.js

```diff
@@ -33,11 +34,13 @@ FROM tests t
     JOIN assertions a ON a.id = ar.assertion_id
     JOIN comparison_types cp ON cp.id = a.comparison_type_id
 
-WHERE tr.id = ${runId}
+WHERE 1=1
```

Comment:
Always true statement added so that the option conditions will always be `AND ...`